src/AccessibilityInsights/Modes/ContainedDialogModeControl.xaml.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.CommonUxComponents.Controls;
+using AccessibilityInsights.SharedUx.Dialogs;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Automation.Peers;
+using System.Windows.Controls;
+
+namespace AccessibilityInsights.Modes
+{
+    /// <summary>
+    /// Interaction logic for ContainedDialogModeControl.xaml
+    /// </summary>
+    public partial class ContainedDialogModeControl : UserControl
+    {
+        public ContainedDialogModeControl()
+        {
+            InitializeComponent();
+        }
+
+        /// <summary>
+        /// Override LocalizedControlType
+        /// </summary>
+        /// <returns></returns>
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            return new CustomControlOverridingAutomationPeer(this, "page");
```

Comment:
how about pulling out "page" string into resource file? 
since "page" is the localized control type, it makes sense to be a resource string.

Reply:
page pulled out here & in other mode controls